packages/pangea-node-sdk/src/services/audit.ts

```diff
@@ -23,11 +23,14 @@ class AuditService extends BaseService {
   private prevUnpublishedRootHash: string | undefined;
   tenantID: string | undefined;
 
-  constructor(token: string, config: PangeaConfig, tenantID: string | undefined = undefined) {
-    super("audit", token, config);
+  constructor(token: string, config: PangeaConfig, tenantID?: string, configID?: string) {
+    // FIXME: Temporary check to still support configID from PangeaConfig
+    if (configID === undefined && config.configID !== undefined) {
```

Comment:
This would allow setting an empty string, unless that's intended, the logic could be:
`if (!configID && !!config.configID)`

Reply:
fixed